spec/controllers/api/plate_controller_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
```

Comment:
Alot of the v2 tests live within this https://github.com/sanger/sequencescape/tree/e75001f3e3a487e98cde90dbfe4b8508af63617d/spec/requests/api/v2 folder too, so it might be worth adding tests there aswell